src/screens/ShowDetailsScreen.tsx

```diff
@@ -153,9 +154,8 @@ const ShowDetailsScreen: React.FC = () => {
         return <ShowDetailsLoader />;
     }
 
-    // TODO: #691 Handle case when no details are ever returned
     if (!details) {
-        return <p>No details found!</p>;
+        return <EmptyShowDetailsScreen />;
```

Comment:
If we're treating this like a component, should we restructure where the file is placed and its name? Instead of screen.

Reply:
I don't think so. This is not being rendered within the show details screen, it is the entire screen when no details are returned.